src/test/mir-opt/address-of.rs

```diff
@@ -35,78 +37,11 @@ fn address_of_reborrow() {
 }
 
 // The normal borrows here should be preserved
+// EMIT_MIR rustc.borrow_and_cast.SimplifyCfg-initial.after.mir
```

Comment:
The previous tests were checking for `rustc.borrow_and_cast.EraseRegions.after.mir`

Reply:
nevermind, `EraseRegions` is gone